sw/host/opentitanlib/src/io/emu.rs

```diff
@@ -35,6 +37,35 @@ pub enum EmuValue {
     FilePathList(Vec<String>),
```

Comment:
Why isn't this PathBuf?

Reply:
I'll check - If pathBuf is serializable i will replace it.
This data go through network by proxy protocol.